content/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/optimize-queries.md

```diff
@@ -87,11 +87,3 @@ If the table contains 10 columns, the difference in performance between the
 two queries is minimal.
 In a table with over 1000 columns, the `SELECT *` query is slower and
 less efficient.
-
-## Analyze and troubleshoot queries
-
-Use the following tools to analyze and troubleshoot queries and find performance bottlenecks:
-
-- [Analyze a query plan](/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/analyze-query-plan/)
-- [Enable trace logging for a query](/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/trace/)
-- [Retrieve `system.queries` information for a query](/influxdb/cloud-dedicated/query-data/troubleshoot-and-optimize/system-information/)
```

Comment:
Should we still leave "Analyze a query plan" and "Retrieve `system.queries` information for a query? Dedicated users should still be able to do these.

Reply:
> Should we still leave "Analyze a query plan" and "Retrieve `system.queries` information for a query? Dedicated users should still be able to do these.

I kept them, just moved them to `troubleshoot.md`. Overall, though, I'm still not happy with the structure/organization in `troubleshoot-and-optimize`.

Reply:
@sanderson It's a little better now, I think. I'll need to port some structure changes to Clustered and Serverless.